src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1138,13 +1141,28 @@ export function makeAxisDataPromise_Molecular_MakeMutationData(
             // we have mutations
             switch (mutationCountBy) {
                 case MutationCountBy.MutationType:
+                    const sampleMutTypes = _.uniq(
+                        sampleMutations.map(
+                            m =>
+                                mutationTypeToDisplayName[
+                                    getOncoprintMutationType(m)
+                                ]
+                        )
+                    );
                     // if more than one type, its "Multiple"
                     if (sampleMutTypes.length > 1) {
                         value = MUT_PROFILE_COUNT_MULTIPLE;
                     } else {
                         value = sampleMutTypes;
                     }
                     break;
+                case MutationCountBy.DriverVsVUS:
+                    if (_.some(sampleMutations, m => m.putativeDriver)) {
+                        value = MUT_PROFILE_COUNT_DRIVER;
```

Comment:
i know this is existing code but would be nice to have a more descriptive nam for "value"

Reply:
good idea. I'll do that before merging